scanomatic/models/settings_models.py

```diff
@@ -195,16 +195,22 @@ def __init__(
         scanner_names: Optional[Sequence[str]] = None,
         scanner_sockets: Optional[dict[str, int]] = None,
     ):
+        assert power_manager is not None
```

Comment:
To me it looks a bit odd to type the class to take optional attributes as input and then directly raise if they are None. Why not just skip the optional typing?

Reply:
See my answer to @skymandr  above